src/builtins/filters/common.rs

```diff
@@ -5,6 +5,7 @@ use std::iter::FromIterator;
 use crate::errors::{Error, Result};
 #[cfg(feature = "builtins")]
 use chrono::{DateTime, FixedOffset, NaiveDate, NaiveDateTime, Utc};
+use chrono_tz::Tz;
```

Comment:
you need to add another `#[cfg(feature = "builtins")]` above that line

Reply:
@Keats Ah, thanks! I was wondering what was causing CI to fail despite it building locally.